tests/test_simple.py

```diff
@@ -1,12 +1,38 @@
 """Tests for mousebender.simple."""
-import importlib.resources
```

Comment:
Use [importilb-resources](https://pypi.org/project/importlib-resources/) instead of rewriting for pathlib.

Reply:
Awesome. I didn't see that! Will try to hunt more aggressively next time.